projects/storefrontlib/src/cms-components/checkout/components/review-submit/review-submit.component.html

```diff
@@ -88,10 +88,14 @@ <h4 class="cx-review-cart-heading d-block d-lg-none d-xl-none">
       class="cx-review-cart-item col-md-12"
       *ngIf="entries$ | async as entries"
     >
+      <ng-container *ngIf="orderPromotions$ | async as orderPromotions">
```

Comment:
Will it add a new markup to existing component for the same scenario? If yes, it should be guarded by a feature flag, otherwise it's considered a breaking.